### PR TITLE
fix issue #792

### PR DESCRIPTION
**Pull request by blurymind**
This fixes this issue:
https://github.com/4ian/GDevelop/issues/792

Sorry that it got to a build. The bug only occurred on builds, this took ages to test, but it works always now.

### CODE REVIEW

File: newIDE/app/public/external/piskel/piskel-main.js [hunk 1]
```diff
@@ -26,15 +26,19 @@ const closeWindow = () => {
 
 // Repeatedly try to gain access to piskel's control element and its methods
 // When succeeding, stop trying.
+const editorFrameEl = document.getElementById('piskel-frame');
+editorFrameEl.src = 'piskel-editor/index.html';
```
Comment (4ian) [inline newIDE/app/public/external/piskel/piskel-main.js]: To be extra cautious, I think we should set the ".src" AFTER registering the onload (to be sure that document is not loaded before the onload is attached - very unlikely but we never now).

Comment (blurymind) [reply newIDE/app/public/external/piskel/piskel-main.js]: ah yes, you're right. Updating now

**Comment by 4ian** [comment]
Thanks! :D

**Comment by blurymind** [comment]
hey no problem, sorry this tainted the release - we need to get it out asap- as without the fix piskel is not very usable

**Comment by 4ian** [comment]
That's ok, I disabled the update soon after it was reported so only a few users got affected + with the new release everything should be back all right :)

**Pull request merged**